### PR TITLE
add vgg network

### PR DESCRIPTION
**Pull request by littletomatodonkey**
Att. remove names

eval acc is as follows.

```
VGG11: {'top1_acc': 0.6929000002861023}
VGG13: {'top1_acc': 0.7003000001716614}
VGG16: {'top1_acc': 0.7200600007820129}
VGG19: {'top1_acc': 0.725639999885559}
```

which is same as before.

### CODE REVIEW

File: ppcls/arch/backbone/legendary_models/vgg.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                kernel_size=3,
+                stride=1,
+                padding=1,
+                bias_attr=False)
+
+        self._pool = MaxPool2D(kernel_size=2, stride=2, padding=0)
+
+    def forward(self, inputs):
+        x = self._conv_1(inputs)
+        x = F.relu(x)
```
Comment (RainFrost1) [inline ppcls/arch/backbone/legendary_models/vgg.py]: `F.relu`尽量替换成`nn.ReLU()`吧，这样替换网络层就能匹配到了,下同

File: ppcls/arch/backbone/legendary_models/vgg.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        self._fc2 = Linear(4096, 4096)
+        self._out = Linear(4096, class_num)
+
+    def forward(self, inputs):
+        x = self._conv_block_1(inputs)
+        x = self._conv_block_2(x)
+        x = self._conv_block_3(x)
+        x = self._conv_block_4(x)
+        x = self._conv_block_5(x)
+        x = x.flatten(start_axis=1, stop_axis=-1)
```
Comment (RainFrost1) [inline ppcls/arch/backbone/legendary_models/vgg.py]: `nn.Flatten()`

File: ppcls/arch/backbone/legendary_models/vgg.py (hunk 1 above)
Comment (littletomatodonkey) [reply ppcls/arch/backbone/legendary_models/vgg.py]: done, thx

File: ppcls/arch/backbone/legendary_models/vgg.py (hunk 2 above)
Comment (littletomatodonkey) [reply ppcls/arch/backbone/legendary_models/vgg.py]: done, thx

**Review by RainFrost1 — APPROVED** [review]

**Review by RainFrost1 — APPROVED** [review]

**Pull request merged**